src/OpenTelemetry.OpAmp.Client/Internal/OpAmpClientEventSource.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics.Tracing;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.OpAmp.Client.Internal;
+
+[EventSource(Name = "OpenTelemetry-OpAmp-Client")]
+internal class OpAmpClientEventSource : EventSource
+{
+    public static OpAmpClientEventSource Log = new();
+
+    // General errors 1-499
+
+    // Service errors 500-999
```

Comment:
At least 2 following events does not looks like errors.
Could you please check all comments?

Reply:
renamed to events.